packages/jest-jasmine2/src/setup-jest-globals.js

```diff
@@ -0,0 +1,80 @@
+/**
+ * Copyright (c) 2014-present, Facebook, Inc. All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @flow
+ */
+
+'use strict';
+
+import type {Config, Path} from 'types/Config';
+
+const {getState, setState} = require('jest-matchers');
+const {initSnapshotState} = require('jest-snapshot');
+
+// get suppressed errors form jest-matchers that weren't throw during the
+// test execution and add them to jasmine result (potentially failing a
+// passing test)
+const addSuppressedErrors = result => {
+  const {notThrownErrors} = getState();
```

Comment:
I think it is usually bad form to call something `notThrownErrors` and inverting the name usually works really well. Like `silencedErrors` or `ignoredErrors` or `errorQueue` or something like it. Anything that doesn't start with `not` :D